base/deprecated.jl

```diff
@@ -2801,6 +2801,28 @@ function Filesystem.tempname(uunique::UInt32)
 end
 end
 
+#25395 keywords unlocked
+@deprecate method_exists(f, t, world) method_exists(f, t, world = world)
+@deprecate Timer(timeout, repeat) Timer(timeout, interval = repeat)
+@deprecate Timer(callback, delay, repeat) Time(callback, delay, interval = repeat)
+@deprecate names(m, all) names(m, all = all)
+@deprecate names(m, all, imported) names(m, all = all, imported = imported)
+@deprecate code_native(io, f, types, syntax) code_native(io, f, types, syntax = syntax)
+@deprecate code_native(f, types, syntax) code_native(f, types, syntax = syntax)
+@deprecate eachmatch(re, str, overlap) eachmatch(re, str, overlap = overlap)
+@deprecate matchall(re, str, overlap) matchall(re, str, overlap = overlap)
+@deprecate chop(s, head) chop(s, head = head)
+@deprecate chop(s, head, tail) chop(s, head = head, tail = tail)
+@deprecate tryparse(T::Type{<:Integer}, s, base) tryparse(T, s, base = base)
+@deprecate parse(T::Type{<:Integer}, s, base) parse(T, s, base = base)
+@deprecate dropzeros(x, trim) dropzeros(x, trim = trim)
+@deprecate dropzeros!(x, trim) dropzeros!(x, trim = trim)
```

Comment:
`dropzeros` and `dropzeros!` should go into `stdlib/SparseArrays/src/deprecate.jl`